cheddar/cheddar-integration-aws/build.gradle

```diff
@@ -2,21 +2,22 @@ apply from: '../../test.gradle'
 apply from: '../../logging-api.gradle'
 
 def springVersion = '4.1.1.RELEASE'
+def awsSdkVersion = '1.11.6'
```

Comment:
I need to ensure that the Cheddar bump ensures that everything is on the latest AWS SDK.